APE14.rst

```diff
@@ -269,21 +284,42 @@ low-level API recommends:
             """
             A dictionary with each key being a string key from
             ``world_axis_object_components``, and each value being a tuple with
-            two elements:
+            two or three elements:
```

Comment:
Why not just mandate this be three long, and just specify the second argument as an empty tuple? It should make all the logic much easier?

Reply:
I agree - someone (maybe @eteq) said they found the empty tuple to not be elegant - but I agree it's additional complication for not much.

Reply:
I don't remember if it was me or not at the time... But me *right bow* agrees with @cadair :wink:. I think in implementation it should be a `NamedTuple` but that doesn't need to be said in the APE.

Reply:
I was about to suggest namedtuple until I remembered the C-API thing, you would have to import and construct the namedtuple instance from where it was created.

Reply:
Fixed in https://github.com/astropy/astropy-APEs/pull/45/commits/e88f4b37f22084021a93b96f61b59c45316f14a3